### PR TITLE
Change to new share path with the production backend dlls 

### PR DESCRIPTION
**Pull request by cindylovescoding**
Change to new share path with the production backend dlls

### CODE REVIEW

File: ApplensBackend/LocalDevelopmentTemplate/StartUp.cmd [hunk 1]
```diff
@@ -2,4 +2,5 @@ Powershell.exe Unblock-File .\Detector\*
 Powershell.exe Unblock-File .\Framework\Tools\*
 Powershell.exe Unblock-File .\Framework\Tools\References\Microsoft.IdentityModel.Clients.ActiveDirectory\*
 Powershell.exe Unblock-File .\Framework\Tools\References\Newtonsoft.Json\*
-powershell.exe -ExecutionPolicy Unrestricted -File .\Framework\Tools\StartUp.ps1
\ No newline at end of file
+powershell.exe -ExecutionPolicy Unrestricted -File .\Framework\Tools\StartUp.ps1
+PAUSE
```
Comment (ShekharGupta1988) [inline ApplensBackend/LocalDevelopmentTemplate/StartUp.cmd]: wanted to remove the PAUSE ?

Comment (cindylovescoding) [reply ApplensBackend/LocalDevelopmentTemplate/StartUp.cmd]: Shekhar I have noticed that even when there is a terminate error, the window will dismiss if we launch the startup.cmd by double click it. Do you think we should keep the window open after the execution finished until user press any key?

Comment (ShekharGupta1988) [reply ApplensBackend/LocalDevelopmentTemplate/StartUp.cmd]: I understand your point. Maybe Pause it only when there is an error.
Having them to press Enter for success case also seems like an extra step.

Comment (cindylovescoding) [reply ApplensBackend/LocalDevelopmentTemplate/StartUp.cmd]: Yeah I was not able to find a decent good way to do that previously.. Let's remove that for now, and add the check back once I have a solution

**Pull request merged**